.vscode/settings.json

```diff
@@ -24,7 +24,7 @@
     "editor.defaultFormatter": "esbenp.prettier-vscode"
   },
   "[typescript]": {
-    "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
Was this change accidental? prettier-plugin-eslint` uses eslint to format, so vscode-eslint should be the 'formatter'.

Reply:
I didn't mean to add this change, it might've slipped in accidentally. I'll revert it back

```suggestion
    "editor.defaultFormatter": "dbaeumer.vscode-eslint"
```